apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/charSequenceAssertions.kt

```diff
@@ -198,6 +198,15 @@ fun <T : CharSequence> Expect<T>.isNotEmpty() = addAssertion(ExpectImpl.charSequ
  */
 fun <T : CharSequence> Expect<T>.isNotBlank() = addAssertion(ExpectImpl.charSequence.isNotBlank(this))
 
+/**
+ * Expects that the subject of the assertion (a [CharSequence]) matches a given [Regex].
```

Comment:
please make sure to mention that it has to match entirely, e,g. use `matches the given [expected] Regex entirely` and I am happy.